.github/workflows/ci.yml

```diff
@@ -20,11 +20,15 @@ jobs:
           node-version: ${{ matrix.node-version }}
       - run: npm install
       - run: npm run lint
-      - env:
-          AWS_REGION: ${{ secrets.AWS_REGION }}
-          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-        run: npm run test:ci
+      - name: configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: ${{ vars.AWS_ROLE_TO_ASSUME }}
+          aws-region: ${{ vars.AWS_REGION }}
+      - run: npm run test:ci
+    permissions:
+      id-token: write
+      contents: read
```

Comment:
as a convention, move the permissions at the beginning of the step please

Reply:
Just did it. 👍🏼

Reply:
Moved it to the same location used in the `automerge` job